frontend/src/components/Pages/FrontPage/MissionOverview/ScheduleMissionDialog.tsx

```diff
@@ -79,7 +83,8 @@ export const ScheduleMissionDialog = (props: IProps): JSX.Element => {
                     disabled={props.frontPageScheduleButtonDisabled}
                     ref={anchorRef}
                 >
-                    {Text('Schedule mission')}
+                    <Icon name="add" size={16} />
```

Comment:
Maybe we also should use Enums in files we only have one icon?

Reply:
https://github.com/orgs/equinor/projects/85/views/4?pane=issue&itemId=22119947